Gruntfile.js

```diff
@@ -66,7 +66,24 @@ module.exports = function (grunt) {
                     protocol: 'https'
                 }
             }
-        }
+        },
+        replace: {
+            dist: {
+              options: {
+                patterns: [
+                  {
+                    match: 'RTC_JS_VERSION',
+                    replacement: '<%= pkg.version %>'
```

Comment:
Wouldn't the change to increase the pkg.version also be part of this commit change.

Reply:
This is my understanding after reading the scripts in package.json. I’m going to merge my changes to master branch. then I switch to bumpVersion branch based on master branch, then increase version number, and build those release files, and create a pr to merge the version number change and new release files from bumpVersion branch to master branch (edited)

Reply:
the short answer is the pkg.version upgrade will be a different pr, along with the new 3 build files